A3A/addons/scrt/Stringtable.xml

```diff
@@ -7783,7 +7783,7 @@
                 <Russian>Направляйтесь на предпологаемое место крушения</Russian>
             </Key>
             <Key ID="STR_A3A_Missions_LOG_crashsite_task_desc">
-                <Original>%1 helicopter with some valueable cargo has been shot down in %2. Bring helicopter's cargo to the HQ before enemy's rescue team. We must get it before %3. Enemy frontline troops already found the crashsite and guarding it.</Original>
+                <Original>%1 The reconnaissance vehicle was shot down. Go to the supposed crash site in %2. Collect the data and bring it to the HQ before enemy rescue team. We must get it before %3. Enemy frontline troops are on their way.</Original>
```

Comment:
Won't this read:
"Faction The reconnaissance vehicle was shot down."?

Reply:
probably yes